### PR TITLE
Add dbm lookup plugin

### PR DESCRIPTION
**Pull request by stoned**
##### SUMMARY
Pretty simple new lookup plugin to fetch values from DBM files.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
lookup plugin dbm

##### ANSIBLE VERSION
```
ansible 2.7.0.dev0 (dbm-lookup b6b20383ea) last updated 2018/06/12 22:45:50 (GMT +200)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/seb/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /u/seb/ghq/github.com/stoned/ansible/lib/ansible
  executable location = /u/seb/ghq/github.com/stoned/ansible/bin/ansible
  python version = 2.7.15 (default, May 16 2018, 17:50:09) [GCC 8.1.1 20180502 (Red Hat 8.1.1-1)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/plugins/lookup/dbm.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# (c) 2018 Stoned Elipot <stoned.elipot(at)gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: dbm
+    author: Stoned Elipot <stoned.elipot(at)gmail.com>
+    version_added: "2.6"
```
Comment (bcoca) [inline lib/ansible/plugins/lookup/dbm.py]: 2.7 at this point

**Review by bcoca — CHANGES_REQUESTED** [review]

File: lib/ansible/plugins/lookup/dbm.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- debug: msg="{{ lookup('dbm','key1', 'key2', file='/tmp/my.dbm') }}"
+"""
+
+RETURN = """
+  _list:
+    description:
+      - values of keys.
+    type: list
+"""
+import anydbm
```
Comment (bcoca) [inline lib/ansible/plugins/lookup/dbm.py]: in py3 this is dbm, so try/except

**Review by bcoca — CHANGES_REQUESTED** [review]

**Review by bcoca — APPROVED** [review]

**Comment by mattclay** [comment]
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/70120/32/tests

**Comment by mattclay** [comment]
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/72512/35/tests

**Comment by mattclay** [comment]
CI failure in integration tests: https://app.shippable.com/github/ansible/ansible/runs/73460/35/tests

**Comment by resmo** [comment]
build failed, this needs investigation before we could merge.

**Comment by Akasurde** [comment]
Hi @stoned, Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/community.general<br><br>If you have further questions please stop by IRC or the mailing list:<br><br>* IRC: #ansible on irc.freenode.net<br>* mailing list: https://groups.google.com/forum/#!forum/ansible-project<br><br>needs_info<br>

**Pull request closed without merging**